{"name": "Проверка невозможности принять заказ при отсутствии ID заказа", "status": "passed", "start": 1732976695862, "stop": 1732976696052, "uuid": "e09188af-e642-4fbd-9ce0-2b98f17691ed", "historyId": "b5cbc6f7212578cf941011a7a464b885", "testCaseId": "b5cbc6f7212578cf941011a7a464b885", "fullName": "tests.test_take_order.TestTakeOrder#test_take_order_without_order_id_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_take_order"}, {"name": "subSuite", "value": "TestTakeOrder"}, {"name": "host", "value": "DESKTOP-4GFTOQP"}, {"name": "thread", "value": "10124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_take_order"}]}